include/substrait-mlir/Dialect/Substrait/IR/CMakeLists.txt

```diff
@@ -1,20 +1,21 @@
 add_mlir_dialect(SubstraitOps substrait)
 add_dependencies(MLIRSubstraitDialect MLIRSubstraitOpsIncGen)
 
+# Add SetOpKind attribute
+set(LLVM_TARGET_DEFINITIONS SubstraitOps.td)
+mlir_tablegen(SubstraitEnums.td.inc -gen-enum-decls)
```

Comment:
```suggestion
mlir_tablegen(SubstraitEnums.h.inc -gen-enum-decls)
```

Reply:
Can we call the file like this and fix the name in the `#include` directive instead?

Reply:
Ah now I understand, thanks!